### PR TITLE
legacy transformations

### PR DESCRIPTION
**Pull request by ilyachur**

### CODE REVIEW

File: inference-engine/src/legacy_api/src/ngraph_ops/eltwise.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 #include "ngraph/util.hpp"
 #include "ngraph/validation_util.hpp"
+#include "../itt.hpp"
```
Comment (ilya-lavrenov) [inline inference-engine/src/legacy_api/src/ngraph_ops/eltwise.cpp:12]: why do we need `..`? I don't see `itt.hpp` inside `legacy_api` folder..

Comment (ilyachur) [reply inference-engine/src/legacy_api/src/ngraph_ops/eltwise.cpp:12]: It is not ready. For some reasons `"itt.hpp` doesn't work. I didn't investigate it but of course I will fix it before the merge.

**Comment by ilya-lavrenov** [comment]
I suppose we can close this?

**Comment by ilyachur** [comment]
Yes

**Pull request closed without merging**